swagger-ga4gh-client/pom.xml

```diff
@@ -7,150 +7,107 @@
+  </dependencies>
   <build>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.12</version>
-        <configuration>
-          <systemProperties>
-            <property>
-              <name>loggerPath</name>
-              <value>conf/log4j.properties</value>
-            </property>
-          </systemProperties>
-          <argLine>-Xms512m -Xmx1500m</argLine>
-          <parallel>methods</parallel>
-          <forkMode>pertest</forkMode>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <executions>
-          <execution>
-            <phase>package</phase>
-            <goals>
-              <goal>copy-dependencies</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${project.build.directory}/lib</outputDirectory>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-
-      <!-- attach test jar -->
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-jar-plugin</artifactId>
-        <version>2.2</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>jar</goal>
-              <goal>test-jar</goal>
-            </goals>
-          </execution>
-        </executions>
-        <configuration>
-        </configuration>
-      </plugin>
-
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>build-helper-maven-plugin</artifactId>
-        <version>1.10</version>
+        <groupId>io.swagger</groupId>
+        <artifactId>swagger-codegen-maven-plugin</artifactId>
+        <version>${swagger-core-version}</version>
         <executions>
           <execution>
-            <id>add_sources</id>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>add-source</goal>
-            </goals>
-            <configuration>
-              <sources>
-                <source>src/main/java</source>
-              </sources>
-            </configuration>
-          </execution>
-          <execution>
-            <id>add_test_sources</id>
-            <phase>generate-test-sources</phase>
             <goals>
-              <goal>add-test-source</goal>
+              <goal>generate</goal>
             </goals>
             <configuration>
-              <sources>
-                <source>src/test/java</source>
-              </sources>
+              <inputSpec>https://raw.githubusercontent.com/ga4gh/workflow-execution-service-schemas/${WES-schemas.version}/openapi/workflow_execution_service.swagger.yaml</inputSpec>
+              <language>java</language>
+              <configOptions>
+                <sourceFolder>src/gen/java/main</sourceFolder>
+                <library>jersey2</library>
+                <dateLibrary>java8</dateLibrary>
```

Comment:
You have java.version 7 above, but you are specifying a dateLibrary of java8.

I don't know what the options are, but it seems like a potential problem. Should at least be researched.

Reply:
https://github.com/swagger-api/swagger-codegen/issues/5863
Suggest that is only a name for the dateLibrary but it does not necessarily support
SDK8 java classes.